src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2662,6 +2662,19 @@ export class StudyViewPageStore {
         default: []
     });
 
+    private genePanelsCache: { [genePanelId: string]: GenePanel } = {};
```

Comment:
I think caching is not required as browser already caches it.

Reply:
Ah, right. The design was a bit different when I introduced the cache. Good catch, let me remove.